### PR TITLE
Data loading for nodes and cabs

### PR DESCRIPTION
**Pull request by juanF18**
Added data loading and data conversion to a python class, also changed some names that were in Spanish.

### CODE REVIEW

**Comment by juanF18** [comment]
[Agregar la funcion para cargar nodos y Taxis desde JSON](https://trello.com/c/wmigm88u/9-agregar-la-funcion-para-cargar-nodos-y-taxis-desde-json)

File: Models/Cab.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        cabs = []
+        for cab_data in data["taxis"]:
+            cab = Cab(
+                cab_data["placa"],
+                cab_data["capacidad_tanque"],
+            )
+            cabs.append(cab)
+        return cabs
+
+    def obtenerPlaca(self):
```
Comment (camian0) [inline Models/Cab.py]: change name to english

File: Models/Cab.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        return self.placa
+
+    def asignarPlaca(self, unaPlaca: str):
```
Comment (camian0) [inline Models/Cab.py]: here

File: Models/Cab.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        if unaPlaca is not None:
+            self.placa = unaPlaca
+
+    def obtenerCapacidad(self):
```
Comment (camian0) [inline Models/Cab.py]: here

File: Models/Cab.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        return self.capacidadTanque
+
+    def asignarCapacidad(self, unaCapacidad: int):
```
Comment (camian0) [inline Models/Cab.py]: here

**Review by camian0 — CHANGES_REQUESTED** [review]

File: Models/Cab.py (hunk 1 above)
Comment (juanF18) [reply Models/Cab.py]: changed in 7a929ff

File: Models/Cab.py (hunk 2 above)
Comment (juanF18) [reply Models/Cab.py]: changed in 7a929ff

File: Models/Cab.py (hunk 3 above)
Comment (juanF18) [reply Models/Cab.py]: Changed in 7a929ff

File: Models/Cab.py (hunk 4 above)
Comment (juanF18) [reply Models/Cab.py]: Changed in 7a929ff

**Review by camian0 — APPROVED** [review]

**Pull request merged**